Library/Homebrew/dev-cmd/boneyard-formula-pr.rb

```diff
@@ -0,0 +1,104 @@
+# Creates a pull request to boneyard a formula.
+#
+# Usage: brew boneyard-formula-pr [options...] <formula-name>
+#
+# Options:
+#   --dry-run:  Print what would be done rather than doing it.
+
+require "formula"
+require "utils/json"
+require "fileutils"
+
+module Homebrew
+  def boneyard_formula_pr
+    formula = ARGV.formulae.first
+    odie "No formula found!" unless formula
+
+    formula_path = formula.path
+    core = formula_path.to_s.include? "homebrew-core"
+    if ARGV.dry_run?
+      ohai "brew update"
```

Comment:
Might use `puts` instead of `ohai` for the dry run output. `ohai` is more of a header. And `ohai` truncates at the terminal's edge instead of wrapping. For debugging output like this you might want to make sure the whole output is available for examination, and without the `==>` prefix.